### PR TITLE
[readme] update quicklinks

### PR DESCRIPTION
**Pull request by uNiversaI**
Updates quicklinks

1) uses kodi.tv
2) new addresses for content

@wsnipex

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,11 +58,12 @@ You can easily install other software on it in addition to Kodi.
 
 ***
 
-##### Quick Kodibuntu development links
+##### Quick Kodiubuntu development links
```
Comment (AlwinEsch) [inline README.md]: Is the name changed to Kodiubuntu?

Comment (wsnipex) [inline README.md]: nope, kodibuntu

**Comment by uNiversaI** [comment]
oops. all fixed now. Plus a  couple of typos the spell checker pointed out.

**Comment by wsnipex** [comment]
thanks

**Pull request merged**